src/api/middleware/validators/auth.js

```diff
@@ -3,19 +3,19 @@ const { body } = require("express-validator");
 const { useExpressValidators } = require("../errorHandler");
 const Account = require("../../../models/Account");
 
-const checkDuplicateUsername = async (username) => {
-    const acc = await Account.findOne({ username }).exec();
+const checkDuplicatedEmail = async (email) => {
+    const acc = await Account.findOne({ email }).exec();
     if (acc) {
-        throw new Error("Username already exists");
+        throw new Error("Email already exists");
     }
 };
 
 const register = useExpressValidators([
-    body("username", "Invalid username")
-        .exists().withMessage("Username is required").bail()
-        .isString().withMessage("Username must be a String")
+    body("email", "Invalid email")
+        .exists().withMessage("Email is required").bail()
+        .normalizeEmail().isEmail().withMessage("Email must be valid")
```

Comment:
This should be using the validator reasons like in the other endpoint validators. The messages don't matter as their content will be decided in the Front-End. As such, we are using enums in the validators.